Data-Structures/src/main/java/datastructures/tree/BinaryTree.java

```diff
@@ -0,0 +1,53 @@
+package datastructures.tree;
+
+import java.util.ArrayList;
+
+public class BinaryTree {
+        ArrayList<Integer> preOrder = new ArrayList<>();
+//        int [] preOrderArray;
+        ArrayList<Integer> inOrder = new ArrayList<>();
+        ArrayList<Integer> postOrder = new ArrayList<>();
+
+        public ArrayList<Integer> preOrder(Node runner){  //left , me, right
+            if(runner != null) {
+//            System.out.println(runner); // prints me.
+                preOrder.add(runner.value);
+//            System.out.println("preOrder is " + preOrder);
+                preOrder(runner.leftOffSpring); //updates the runner to leftoffspring.
+                preOrder(runner.rightOffSpring); //updates runner to right;
+            }
+//            Object [] preOrderArray = preOrder.toArray();
+
+//        }else{
+//            return; // do nothing. go to sleep.
+//        }
+//            return preOrderArray;
```

Comment:
You have a lot of commented out code, which is fine if you're still working on it. But you should start getting in the habit of cleaning it up before you push up your code. It makes your code difficult to read, and you can always go back to a previous commit if you need to.